File: docs/source/_src_snippets/simulation/environment_setup/environment_models/interpolated_spice_ephemeris.py
initial_time = 0.0
final_time = 1.0E8
time_step = 3600.0

frame_origin = "SSB"
frame_orientation = "J2000"

body_settings.get_body( "Jupiter" ).ephemeris_settings = environment_setup.ephemeris.interpolated_spice(
	initial_time, final_time, time_step, frame_origin, frame_orientation )

File: docs/source/_src_snippets/simulation/environment_setup/environment_models/interpolated_spice_ephemeris.cpp
double initialTime = 0.0;
double finalTime = 1.0E8;
double timeStep = 3600.0;

std::string frameOrigin = "SSB";
std::string frameOrientation = "J2000";

bodySettings[ "Jupiter" ]->ephemerisSettings = std::make_shared< InterpolatedSpiceEphemerisSettings >(
    initialTime, finalTime, timeStep, frameOrigin, frameOrientation );